build/yamls/antrea-eks.yml

```diff
@@ -174,12 +174,72 @@ spec:
     shortNames:
     - tf
     singular: traceflow
+  preserveUnknownFields: false
   scope: Cluster
+  subresources:
+    status: {}
   validation:
     openAPIV3Schema:
       properties:
         spec:
           properties:
+            destination:
+              oneOf:
+              - required:
+                - pod
+                - namespace
+              - required:
+                - ip
```

Comment:
should this be `IP`?

Reply:
I mean it's "IP" in spec: https://github.com/vmware-tanzu/antrea/blob/master/pkg/apis/ops/v1alpha1/types.go#L84.
But according to the convention, it should be "ip". See a similar case: https://github.com/kubernetes/kubernetes/blob/v1.18.5/staging/src/k8s.io/api/core/v1/types.go#L3124

Reply:
correct.. this should actually be "ip" in our types.go. do you want me to correct it in this PR?

Reply:
addressed in latest commit